projects/ngx-launcher/src/lib/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts

```diff
@@ -46,6 +56,10 @@ export class ProjectProgressCreateappNextstepComponent implements OnChanges, OnD
       const message = JSON.parse(event.data);
       console.log('data from ws', message);
       const data = message.data || {};
+      if (data.codeBaseId !== undefined) {
```

Comment:
where is the backend change to make it send back this information?

Reply:
@bartoszmajsak Can you please point the backend changes for this?